charlib/characterizer/port.py

```
@@ -106,7 +106,26 @@ def __init__(self, noninverting_port_name: str, inverting_port_name: str, direct
         self.noninverting_port_name = noninverting_port_name
         self.inverting_port_name = inverting_port_name
 
+    def __contains__(self, item) -> bool:
+        """Implements 'in' operator"""
+        if isinstance(item, str):
+            return item in [self.noninverting_port_name, self.inverting_port_name]
+        elif isinstance(item, Pin):
```

Ruff (0.15.17)

[warning] Unnecessary `elif` after `return` statement

Remove unnecessary `elif`

(RET505)